src/plugins/discover/public/application/components/context_app/context_app_legacy.tsx

```diff
@@ -17,33 +17,86 @@
  * under the License.
  */
 import React from 'react';
-import { FormattedMessage } from '@kbn/i18n/react';
+import { FormattedMessage, I18nProvider } from '@kbn/i18n/react';
 import { EuiPanel, EuiText } from '@elastic/eui';
-import { I18nProvider } from '@kbn/i18n/react';
 import {
   DocTableLegacy,
   DocTableLegacyProps,
 } from '../../angular/doc_table/create_doc_table_react';
 import { IIndexPattern, IndexPatternField } from '../../../../../data/common/index_patterns';
 import { LOADING_STATUS } from './constants';
+import { ActionBar, ActionBarProps } from '../../angular/context/components/action_bar/action_bar';
 
-export interface ContextAppProps {
+export type ContextAppProps = Partial<ActionBarProps> & {
```

Comment:
Since only `defaultStepSize` of ActionBarProps seems to be used, it might make sense to add this to ContextAppProps to make this explicit and interface it